ludwig/features/category_feature.py

```diff
@@ -419,26 +417,26 @@ def postprocess_predictions(
 
     @staticmethod
     def populate_defaults(output_feature):
+        defaults = CategoryOutputFeatureConfig()
```

Comment:
Is `populate_defaults()` needed anymore? I believe all defaults should be encoded in the schema, so I don't think we need to set them explicitly, but perhaps I'm missing something.

Reply:
So I believe there are a couple of defaults that still need to be set with `populate_defaults()` before it gets to the actual feature initialization, IIRC for the preprocessing step. I will double check on this. Either way we will be removing this once the full config object lands so you are correct that they shouldn't be needed anymore with the new schemas, just think theres a small amount of additional wiring needed to fully remove this.

Reply:
For now, `populate_defaults()` is needed for preprocessing, since we don't instantiate the feature objects until after preprocessing.  We don't need an implementation per class anymore though, because all of the defaults can be stored in the schema.  @connor-mccorm is planning to remove populate_defaults in his config object PR, which will be a follow-up to this one.